### DISCUSSION TITLE
Confused about breaking changes in v102

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#9184] [Q&A]
I am a bit confused with the v102 release.
What does it mean "pulling the latest compose file"?
I never pull the compose file, only modify it when it says so in the release notes.
Does this mean that the next time a change in the compose file is needed there will be progress with the database?

**Comment by alextran1502** [comment]
> What does it mean "pulling the latest compose file"?

It means there are users copy the latest compose file that attached to the release without checking if there is any changes about the content
